graphs/kickiip.fbp

```diff
@@ -1,4 +1,5 @@
-# !runtime noflo
+# @runtime noflo-nodejs
+# @name IIPKickTest
```

Comment:
If the name comes from @name and things can't be in non-standard paths (see previous comment), then whats the point of having data in `package.json` at all?